### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by teythoon**

### CODE REVIEW

**Review by fluppe2 — APPROVED** [review]

File: draft-wussler-openpgp-pqc.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ becomes available.\]
 
 ML-KEM [FIPS-203] is based on the hardness of solving the learning-with-errors
 problem in module lattices (MLWE). The scheme is believed to provide security
-against cryptanalytic attacks by classical as well as quantum computers. This
+against cryptoanalytic attacks by classical as well as quantum computers. This
```
Comment (falko-strenzke) [inline draft-wussler-openpgp-pqc.md:359]: I disagree here, "cryptanalytic" or "cryptanalytis" are the commonly used term with a hits on Google while for "cryptoanalytic" I don't see any results on Google.

File: draft-wussler-openpgp-pqc.md [hunk 2]
```diff
@@ -1465,14 +1465,14 @@ context.
...
-univocally the algorithm, the parameters for its instantiation, and the length
+unequivocally the algorithm, the parameters for its instantiation, and the length
 of all artifacts, including the derived key.
 
 This is in line with the Recommendation for ECC in section 5.5 of
 [SP800-56A]. Other fields included in the recommendation are not relevant
 for the OpenPGP protocol, since the sender is not required to have a key of
 their own, there are no pre-shared secrets, and all the other parameters are
-univocally defined by the algorithm ID.
+unequivocally defined by the algorithm ID.
```
Comment (falko-strenzke) [inline draft-wussler-openpgp-pqc.md:1475]: I think both words mean the same but I am fine with changing it.

File: draft-wussler-openpgp-pqc.md [hunk 3]
```diff
@@ -622,7 +622,7 @@ ECDH component of the composite algorithms.
 
 {{tab-ecdh-cfrg-artifacts}}, {{tab-ecdh-nist-artifacts}}, and
 {{tab-ecdh-brainpool-artifacts}} describe the ECC-KEM parameters and artifact
-lengths. The artefacts in {{tab-ecdh-cfrg-artifacts}} follow the encodings
+lengths. The artifacts in {{tab-ecdh-cfrg-artifacts}} follow the encodings
```
Comment (falko-strenzke) [inline draft-wussler-openpgp-pqc.md:625]: I am not sure whether we have commited to BE or AE. In BE, "artefact" would be appropriate: https://grammarist.com/spelling/artefact-artifact/

File: draft-wussler-openpgp-pqc.md (hunk 1 above)
Comment (dkg) [reply draft-wussler-openpgp-pqc.md:359]: agree with @falko-strenzke here, i tend to think cryptanalytic is the preferred form.

File: draft-wussler-openpgp-pqc.md (hunk 2 above)
Comment (dkg) [reply draft-wussler-openpgp-pqc.md:1475]: agree with @teythoon's  proposed change here.  As a native en_US speaker with a decent vocabulary, unequivocally is a much more common and well-understood word choice.  i can parse "univocally" and figure out what it means ("with one voice") but it's not anything that i'm used to reading.

File: draft-wussler-openpgp-pqc.md (hunk 3 above)
Comment (falko-strenzke) [reply draft-wussler-openpgp-pqc.md:625]: OK, great, thanks for these insights, DKG. Once the topmost change regarding "cryptanalytic" has been reverted, I will hit "approve".

**Review by falko-strenzke — APPROVED** [review]
@teythoon Thanks for these corrections.

**Comment by falko-strenzke** [comment]
@teythoon Could you please still remove the conflicts with the main branch – probably it' s the renaming of the file.

**Pull request merged**